astronomer/providers/google/cloud/example_dags/example_gcs.py

```diff
@@ -13,55 +31,71 @@
     LocalFilesystemToGCSOperator,
 )
 
-from astronomer.providers.google.cloud.sensors.gcs import GCSObjectExistenceSensorAsync
+from astronomer.providers.google.cloud.sensors.gcs import (
+    GCSObjectExistenceSensorAsync,
+    GCSObjectsWithPrefixExistenceSensorAsync,
+    GCSUploadSessionCompleteSensorAsync,
+)
 
-START_DATE = datetime(2022, 1, 1)
+START_DATE = datetime(2021, 1, 1)
```

Comment:
nit: since `START_DATE` is getting used once only we should directly use  `datetime(2021, 1, 1)` in line 51